frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -93,7 +93,6 @@ type Props = {
   formatOptions?: FormattingOptions,
   maxWidth?: number,
   minWidth?: number,
-  optionsMaxHeight?: Number,
```

Comment:
This is mysterious because the only values passed to `optionsMaxHeight` were `null` and `undefined` based on this line `optionsMaxHeight={isSidebar ? null : undefined}` from `frontend/src/metabase/query_builder/components/filters/pickers/DefaultPicker.jsx`